### ISSUE TITLE
Readme error

### ISSUE THREAD

**Issue by fvckDesa** [sinclairzx81/typebox#548]
I try to import "ValueErrorType" from "@sinclair/typebox/system" like in the README file and I recive the follow error:
```text
Module '"@sinclair/typebox/system"' has no exported member 'ValueErrorType'.
```

I found "ValueErrorType" in module "@sinclair/typebox/errors"

**Comment by fvckDesa** [comment]
I take advantage of this issue for a question.
The are some internal bug that I can cause if I use TypeRegistry.Set with existing Kind like "Enum"

**Comment by sinclairzx81** [comment]
@fvckDesa Hi, Thanks for reporting!

> I found "ValueErrorType" in module "@sinclair/typebox/errors"

This looks like an oversight to me (there was some consideration moving errors to the `system` module, but missed re-exporting the `ValueErrorType` on 0.31.0). I'll push an update to fix this today. 

---

### Override Built in Types

> The are some internal bug that I can cause if I use TypeRegistry.Set with existing Kind like "Enum"

It shouldn't be possible to override the built in types (as these are implicitly handled internal to the library). 

```typescript
TypeRegistry.Set('Number', (schema, value) => {
  console.log('Inside Number Kind')
  return true
})

const T = Type.Number()     // { [Kind]: 'Number' }

const R = Value.Check(T, 1) // Does not use registry
```

Note: The `[Kind]` will only run in instances where implicit checks fail to match.  See [here](https://github.com/sinclairzx81/typebox/blob/master/src/value/check.ts#L365-L429) for how TB internally handles Kind matching.

---

### Enum

Enum is actually a expressed as a mapped Union in TypeBox. 

```typescript
const T = Type.Enum({         // const T = {
  x: 1,                       //   anyOf: [
  y: 2                        //     { type: 'number', const: 1, [Symbol(TypeBox.Kind)]: 'Literal' },
} as const)                   //     { type: 'number', const: 2, [Symbol(TypeBox.Kind)]: 'Literal' }
                              //   ],
                              //   [Symbol(TypeBox.Kind)]: 'Union'
                              // }
```
Because of this, it is actually possible to create a custom Enum `[Kind]` (as TypeBox doesn't have an internal concept of Enum, only Union)

```typescript
TypeRegistry.Set('Enum', (schema: any, value) => schema.enum.includes(value))

const StringEnum = <T extends string[]>(values: [...T]) => {
  return Type.Unsafe<T[number]>({ [Kind]: 'Enum', enum: values })
}


const T = StringEnum(['A', 'B', 'C'])   // const T = { 
                                        //   enum: [ 'A', 'B', 'C' ], 
                                        //   [Symbol(TypeBox.Kind)]: 'Enum' 
                                        // }

Value.Check(T, 'A')                     // true
Value.Check(T, 'B')                     // true
Value.Check(T, 'C')                     // true
Value.Check(T, 'D')                     // false
```
Hope this helps!
S

**Comment by sinclairzx81** [comment]
@fvckDesa Hi!

Have submitted a patch to re-export the `ValueErrorType` on the system module (as per documentation). Updates available on `0.31.4`. 

Thanks for reporting! Will close off this issue for now :)
All the best
S
